tests/functional/intelligent-host-selection/04-kill.t

```diff
@@ -17,7 +17,7 @@
 #-------------------------------------------------------------------------------
 # Test job kill will retry on a different host if there is a connection failure
 
-export REQUIRE_PLATFORM='loc:remote fs:indep comms:tcp'
+export REQUIRE_PLATFORM='loc:remote fs:indep comms:tcp runner:background'
```

Comment:
Test relies on mocking background submission (`my_background`) so can only be run with background platforms.